csm/conf/setup.py

```diff
@@ -461,6 +461,12 @@ def _set_rmq_cluster_nodes(self):
                         nodes_found = True
                 if nodes_found:
                     break
+                result = re.search(
```

Comment:
This code is not inside `for line in cmd_output[0].split('\n'):`
Why do we need this for loop and if we need it then the new code should be inside for loop?

Reply:
There are two methods used to fetch RMQ running nodes.

First one is using "for line in cmd_output[0].split('\n'):" to read command output line by line, find strings that contains "running_nodes" substring, and fetch node names.

The second one doesn't use for loop - it fetches running nodes list directly from command output